experimenter/experimenter/experiments/models.py

```diff
@@ -676,6 +676,34 @@ def results_ready(self):
             )
             return datetime.date.today() >= resultsReadyDate
 
+    @property
+    def has_displayable_results(self):
+        # True if self.results_data has weekly or overall results
+        if self.results_data and "v2" in self.results_data:
+            results_data = self.results_data["v2"]
+            if "overall" in results_data:
+                overall_results = results_data["overall"]
+                if "enrollments" in overall_results:
+                    enrollments = overall_results["enrollments"]
+                    if "all" in enrollments and enrollments["all"] is not None:
+                        return True
+
+            if "weekly" in results_data:
+                weekly_results = results_data["weekly"]
+                if "enrollments" in weekly_results:
+                    enrollments = weekly_results["enrollments"]
+                    if "all" in enrollments and enrollments["all"] is not None:
+                        return True
+
+        return False
+
+    @property
+    def show_results_url(self):
+        # TODO: more logic needed? if not then return IF directly
+        if self.has_displayable_results and self.results_ready and not self.is_rollout:
```

Comment:
💃

Reply:
oh you can do `return self.has_displayable_results and self.results_ready and not self.is_rollout`

Reply:
Not sure how I missed it on commit.. whoops sorry! Glad I put this in though.